app/TracingStrategyAssets.ts

```diff
@@ -1,214 +0,0 @@
-import { useTranslation } from 'react-i18next';
-
-import { Icons, Images } from './assets';
-import { isGPS } from './COVIDSafePathsConfig';
-
-export type Asset = string | ((name: string) => string) | null;
-
-export function useAssets(): Record<string, Asset> {
-  const { t } = useTranslation();
-
-  // Onboarding2
-  const onboarding2Background = isGPS
```

Comment:
so the main goal is to remove as many instances of `isGPS` as possible, as this a challenge to maintain and will only get worse as we add more tracing strategies.  So we instead of checking at each instance of some copy, we provide an interface of whole set of strategy specific copy/assets/whatever, expect the tracing strategy to provide that wholesale, and then we only need to check `isGPS` once and provide the all of content as a single group of data.